__tests__/uberLinks.js

```diff
@@ -0,0 +1,31 @@
+import { getUberDeepLink, getUberUniversalLink } from '../components/UberButton/uberLinks';
+
+describe('UberButton helper functions', () => {
+  describe('getUberDeepLink', () => {
+    it('is a function', () => {
+      expect(typeof getUberDeepLink).toBe('function');
+    });
+
+    it('returns a string', () => {
+      expect(typeof getUberDeepLink()).toBe('string');
+    });
+
+    it('returns an uber deep link', () => {
+      expect(getUberDeepLink()).toEqual(expect.stringContaining('uber://'));
+    });
```

Comment:
stringContaining() is only supported in Jest 19+. package.json is updated below.